eland/ml/pytorch/transformers.py

```diff
@@ -769,15 +774,124 @@ def _create_config(
                 tokenization=tokenization_config
             )
 
+        # add static and dynamic memory state size to metadata
+        per_deployment_memory_bytes = self._get_model_memory()
+
+        per_allocation_memory_bytes = self._get_transient_memory(
+            tokenization_config.max_sequence_length, 1
+        )
+        peak_memory_bytes = self._get_peak_memory(
+            per_deployment_memory_bytes, per_allocation_memory_bytes
+        )
+        # TODO: final field names are subject to change
+        metadata = {
+            "per_deployment_memory_bytes": per_deployment_memory_bytes,
+            "per_allocation_memory_bytes": per_allocation_memory_bytes,
+            "peak_memory_bytes": peak_memory_bytes,
+        }
+
+        # TODO: remove this once memory metadata is supported by ES
```

Comment:
Is this TODO outdated as I see `per_deployment_memory_bytes ` and `per_allocation_memory_bytes` used in TrainedModelConfig.java (but no reference to `peak_memory_bytes`)

Reply:
Indeed, I removed the TODO comment and peak_memory_bytes